dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelectorContent.jsp

```diff
@@ -1,74 +1,160 @@
-                        <button dojoType="dijit.form.Button" dojoAttachEvent='onClick:_clearSearch' iconClass="cancelIcon" class="dijitButtonFlat" style="margin-top: 16px"><%= LanguageUtil.get(pageContext, "Clear") %></button>
-                    </div>
+<form
+    dojoAttachPoint="search_form"
+    onsubmit="return false;"
+    id="searchForm"
+    class="related-content-form"
+>
+    <!-- START Left Column -->
+    <div dojoType="dijit.layout.ContentPane" class="portlet-sidebar-wrapper">
+        <div class="portlet-sidebar">
+            <input
+                type="hidden"
+                name="hostField"
+                dojoAttachPoint="hostField"
+                value=""
+            />
+            <input
+                type="hidden"
+                name="folderField"
+                dojoAttachPoint="folderField"
+                value=""
+            />
+            <input
+                type="hidden"
+                name="structure_inode"
+                dojoAttachPoint="structure_inode"
+                value="strInode"
+            />
+            <div class="sideMenuWrapper">
+                <div dojoAttachPoint="structures_select"></div>
+                <div dojoAttachPoint="search_languages_table"></div>
+                <div dojoAttachPoint="search_fields_table"></div>
+                <div dojoAttachPoint="search_categories_table">
+                    <dl
+                        class="vertical"
+                        dojoAttachPoint="search_categories_list"
+                    ></dl>
+                </div>
+                <div dojoAttachPoint="search_general">
+                    <dl class="vertical">
+                        <dt>
+                            <label
+                                ><%= LanguageUtil.get(pageContext, "Search")
+                                %>:</label
+                            >
+                        </dt>
+                        <dd>
+                            <input
+                                type="text"
+                                dojoType="dijit.form.TextBox"
+                                onKeyDown="contentSelector._doSearch()"
+                                dojoAttachPoint="generalSearch"
+                            />
+                        </dd>
+                    </dl>
+                </div>
+                <div class="clear"></div>
+                <div class="buttonRow">
+                    <button
+                        dojoType="dijit.form.Button"
+                        dojoAttachEvent="onClick:_doSearchPage1"
+                        iconClass="searchIcon"
+                    >
+                        <%= LanguageUtil.get(pageContext, "Search") %>
+                    </button>
+                    <button
+                        dojoType="dijit.form.Button"
+                        dojoAttachEvent="onClick:_clearSearch"
+                        iconClass="cancelIcon"
+                        class="dijitButtonFlat"

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5076884)